backends/gstreamer/player.rs

```diff
@@ -550,7 +428,11 @@ impl GStreamerPlayer {
             .set_config(config)
             .map_err(|e| PlayerError::Backend(e.to_string()))?;
 
-        let video_sink = self.setup_video_sink(&player)?;
+        let render = self
+            .render
+            .replace(None)
```

Comment:
I believe you want to use `into_inner` instead of `replace` here.

Reply:
I get this error: 

```
error[E0507]: cannot move out of borrowed content
```